### DISCUSSION TITLE
How to create "New Folder" next to Demo and show in File Manager?

### DISCUSSION THREAD

**Discussion by naveenlb316** [givanz/VvvebJs#367] [General]
I want to create a new folder "New Folder" which will be same level as "Demo" folder. I want to show these contents in File Manager of Editor. Please let me know how to make these changes.

In File Manager, it should show both the contents of Demo and New folder.

**Comment by givanz** [comment]
You need to add the folder to the glob list 

Replace line 7 in [editor.php ](https://github.com/givanz/VvvebJs/blob/master/editor.php#L7) with 

```php
$htmlFiles = array_merge(glob('my-pages/*.html'),glob('demo/*\/*.html'), glob('demo/*.html'), glob('new-folder/*\/*.html'), glob('new-folder/*.html'));
```

You must open the editor using `editor.php` instead of `editor.html` to load new files.
